Mathlib/Algebra/Category/ModuleCat/Monoidal/Basic.lean

```diff
@@ -103,6 +104,15 @@ instance instMonoidalCategoryStruct : MonoidalCategoryStruct (ModuleCat.{u} R) w
   leftUnitor := leftUnitor
   rightUnitor := rightUnitor
 
+attribute [moduleCat_simps] ModuleCat.MonoidalCategory.tensorObj
```

Comment:
I would think `monoidal` should appear in the name `moduleCat_simps` because these lemmas involve the monoidal structure.